zen_queries/decorators.py

```diff
@@ -1,47 +1,56 @@
 from contextlib import contextmanager
 from django.db import connections
+from typing import Any, Callable, Dict, Generator
 
 
 class QueriesDisabledError(Exception):
     pass
 
 
-def _raise_exception(execute, sql, params, many, context):
+def _raise_exception(
+    execute: Callable[[str, Any, bool, Dict[str, Any]], Any],
```

Comment:
Thought: Given we don't care about the value of almost any of these arguments, might it be more maintainable to just type them as `Any`? Given the number of `Any`'s already here

Reply:
I think that would be entirely OK, as this isn't public API and the types here thus doesn't matter for a user of django-zen-queries. At the same time, I don't expect the Django API to change much, and any type-only changes wouldn't break django-zen-queries at runtime.